docs/sources/flow/reference/components/loki.source.api.md

```diff
@@ -25,9 +25,12 @@ loki.source.api "LABEL" {
 
 The component will start HTTP server on the configured port and address with the following endpoints:
 
-- `/api/v1/push` - accepting `POST` requests compatible with [Loki push API][loki-push-api], for example, from another Grafana Agent's [`loki.write`][loki.write] component.
-- `/api/v1/raw` - accepting `POST` requests with newline-delimited log lines in body. This can be used to send NDJSON or plaintext logs. This is compatible with promtail's push API endpoint - see [promtail's documentation][promtail-push-api] for more information. NOTE: when this endpoint is used, the incoming timestamps cannot be used and the `use_incoming_timestamp = true` setting will be ignored. 
-- `/ready` - accepting `GET` requests - can be used to confirm the server is reachable and healthy.
+- `/loki/api/v1/push` - accepting `POST` requests compatible with [Loki push API][loki-push-api], for example, from another Grafana Agent's [`loki.write`][loki.write] component.
+- `/loki/api/v1/raw` - accepting `POST` requests with newline-delimited log lines in body. This can be used to send NDJSON or plaintext logs. This is compatible with promtail's push API endpoint - see [promtail's documentation][promtail-push-api] for more information. NOTE: when this endpoint is used, the incoming timestamps cannot be used and the `use_incoming_timestamp = true` setting will be ignored. 
+- `/loki/ready` - accepting `GET` requests - can be used to confirm the server is reachable and healthy.
+- `/api/v1/push` - Internally reroutes to `/loki/api/v1/push` 
+- `/api/v1/raw` - Internally reroutes to `/loki/api/v1/raw`
```

Comment:
```suggestion
- `/api/v1/push` - internally reroutes to `/loki/api/v1/push` 
- `/api/v1/raw` - internally reroutes to `/loki/api/v1/raw`
```

Just a suggestion for consistency since all others are lower case